src/components/DataCollector/index.js

```diff
@@ -56,9 +57,14 @@ const createDataCollector = ({ eventManager, logger }) => {
         event.setUserXdm(xdm);
         event.setUserData(data);
 
-        return eventManager.sendEvent(event, {
-          isViewStart: viewStart
-        });
+        const details = {};
+        details.isViewStart = viewStart;
```

Comment:
Maybe just initialize the object with the data we know up front:

```
const details = {
  isViewStart: viewStart
};
```

Reply:
yes, thank you